man/man7/fi_opx.7

```diff
@@ -50,11 +50,14 @@ Additional features
 Supported additional features include \f[I]FABRIC_DIRECT\f[R],
 \f[I]scalable endpoints\f[R], and \f[I]counters\f[R].
 .TP
-Progress
-Only \f[I]FI_PROGRESS_MANUAL\f[R] is supported.
+.B Progress
```

Comment:
the change to `fi_opx.7` is unnecessary. The actual man page is going to be automatically generated from the corresponding '.md' file.

Reply:
Is there something in the *.7 that you see that is out of sync with the *.md file? Because it's our understanding that sometimes the *.7 isn't regenerated when it should be, so it's better to be safe and push modifications to it too.

Reply:
We talked about it internally and agree that we should be requesting the regen rather than trying to push this file. I updated the relevant commit and will also do it for the corresponding PR against main.